.github/workflows/_ci.yaml

```diff
@@ -15,18 +15,43 @@ on:
       TAG_BASE:
```

Comment:
can't comment on the line, but looks like you don't use TRIAL_BRANCH on line 10

Reply:
Removed